src/main/java/ghostcat/capstone/image_query/ImageQueryDAO.java

```diff
@@ -72,8 +83,8 @@
 
         String keyExp = "UserID = :v_userID and img_id = :v_imageId";
         ValueMap values = new ValueMap()
-                .with(":v_userID", userID)
-                .with(":v_imageId", imageId);
+                .withString(":v_userID", userID)
+                .withString(":v_img_id", imageId);
```

Comment:
This change needs to be removed, sorry I missed this when fixing the merge conflict